arm-sql/2015-05-01-preview/swagger/resourceDefinitions.json

```diff
@@ -0,0 +1,271 @@
+        },
+        "tier":{  
+          "type":"string",
+          "enum":[  
+            "free",
+            "basic",
+            "Standard",
+            "Premium"
+          ],
+          "description":"This field is required to be implemented by the Resource Provider if the service has more than one tier, but is not required on a PUT."
+        },
+        "size":{  
+          "type":"string",
+          "description":"The SKU size. When the name field is the combination of tier and some other value, this would be the standalone code. "
+        },
+        "family":{  
+          "type":"string",
+          "description":"If the service has different generations of hardware, for the same SKU, then that can be captured here."
+        },
+        "capacity":{  
+          "type":"integer",
+          "format":"int32",
+          "description":"If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted."
+        }
+      },
+      "required":[  
+        "name"
+      ]
+    },
+    "Identity":{  
+      "description":"Identity for the resource.",
+      "properties":{  
+        "principalId":{  
+          "readOnly":true,
+          "type":"string",
+          "description":"The principal ID of resource identity."
+        },
+        "tenantId":{  
+          "readOnly":true,
+          "type":"string",
+          "description":"The tenant ID of resource."
+        },
+        "type":{  
+          "type":"string",
+          "description":"The identity type.",
+          "enum":[  
+            "SystemAssigned"
+          ],
+          "x-ms-enum":{  
+            "name":"ResourceIdentityType",
+            "modelAsString":false
+          }
+        }
+      }
+    },
+    "Plan":{  
+      "properties":{  
+        "name":{  
+          "type":"string",
+          "description":"A user defined name of the 3rd Party Artifact that is being procured."
+        },
+        "publisher":{  
+          "type":"string",
+          "description":"The publisher of the 3rd Party Artifact that is being bought. E.g. NewRelic"
+        },
+        "product":{  
+          "type":"string",
+          "description":"The 3rd Party artifact that is being procured. E.g. NewRelic. Product maps to the OfferID specified for the artifact at the time of Data Market onboarding. "
+        },
+        "promotionCode":{  
+          "type":"string",
+          "description":"A publisher provided promotion code as provisioned in Data Market for the said product/artifact."
+        },
+        "version":{  
+          "type":"string",
+          "description":"The version of the desired product/artifact."
+        }
+      },
+      "description":"Plan for the resource.",
+      "required":[  
+        "name",
+        "publisher",
+        "product"
+      ]
+    }
+  },
+  "parameters":{  
```

Comment:
I was just copying resourceDefinitions.json from Azure/azure-rest-api-specs-pr#40 since I thought that's what you were asking for.